deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: altinity-clickhouse-operator
+description: |-
+  Helm chart to deploy [altinity-clickhouse-operator](https://github.com/Altinity/clickhouse-operator).
+
+  The ClickHouse Operator creates, configures and manages ClickHouse clusters running on Kubernetes.
+type: application
+version: 0.20.0
+appVersion: 0.20.0
+home: https://github.com/Altinity/clickhouse-operator
+icon: https://altinity.com/wp-content/uploads/2022/05/logo_horizontal_blue_white.svg
+maintainers:
+  - name: altinity
+    email: info@altinity.com
```

Comment:
support@altinity.com will better here 
the same as https://operatorhub.io/operator/clickhouse